README.md

```diff
@@ -5,6 +5,10 @@
 [![Puppet Forge - downloads](https://img.shields.io/puppetforge/dt/puppet/lldpd.svg)](https://forge.puppetlabs.com/puppet/lldpd)
 [![Puppet Forge - endorsement](https://img.shields.io/puppetforge/e/puppet/lldpd.svg)](https://forge.puppetlabs.com/puppet/lldpd)
 [![Puppet Forge - scores](https://img.shields.io/puppetforge/f/puppet/lldpd.svg)](https://forge.puppetlabs.com/puppet/lldpd)
+[![Yard Docs](https://img.shields.io/badge/yard-docs-blue.svg)](https://voxpupuli.org/puppet-lldpd)
+[![AGPL v3 License](https://img.shields.io/github/license/voxpupuli/puppet-lldpd.svg)](LICENSE)
+[![Dependencies](https://img.shields.io/gemnasium/voxpupuli/puppet-lldpd.svg)](https://gemnasium.com/voxpupuli/puppet-lldpd)
```

Comment:
This (and Codacy) don't appear that useful for puppet modules.

Reply:
yeah, the gemnasium badge will always be green and codacy will report on the quality of the ruby code in the tests and ignore all of the puppet.